### PR TITLE
Replace LESS with Sass

### PR DESCRIPTION
**Pull request by nicolaisueper**
Now using Sass instead of LESS in our build workflow.

I've also ported our existing LESS files to Sass. ✨

### CODE REVIEW

**Comment by mischah** [comment]
Thanks. Needs to be discussed if we would like to make that switch with the v4 release (before the updating to Bootstrap 4).

cc/ @micromata/ux

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "private": true,
   "dependencies": {
-    "bootstrap": "3.3.7",
+    "bootstrap": "4.0.0-alpha.6",
```
Comment (mischah) [inline package.json]: Please replace Bootstrap 4 with latest 3.x Sass Bootstrap for now.

Comment (nicolaisueper) [inline package.json]: see 356ffde :)

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "babel-preset-env": "^1.2.2",
     "babel-register": "^6.24.1",
     "babelify": "^7.3.0",
+    "bootstrap-sass": "^3.3.7",
```
Comment (mischah) [inline package.json]: Move to dependencies.

Comment (mischah) [inline package.json]: Do we still need `"bootstrap": "3.3.7"` in the dependencies?

Doesn’t ship `bootstrap-sass` all we need?

Comment (nicolaisueper) [inline package.json]: You're right. I thought, bootstrap-sass only ships the styles. Done in 9c5fad2.

**Comment by nicolaisueper** [comment]
merge? ☺️

**Comment by mischah** [comment]
> merge? ☺️

I’m okay with it. But I need an approval from everyone involved.

@micromata/ux:

This PR replaces the LESS version of Bootstrap `3.3.7` with the official Sass port of `3.3.7`
With Bootstrap 4 we need to make the switch from LESS to Sass.  Why not making the switch right now? Approve the PR if your want this to happen with the next release of Bootstrap Kickstart.

**Comment by mischah** [comment]
We get a little, but nice performance upgrade with this:
Transpiling styles while watching is **at least 100ms** faster with Sass.

_LESS:_
image

_Sass:_
image

**Comment by krnlde** [comment]
Nice! libsassc is awesome

**Comment by mischah** [comment]
Merged. Thanks @nicolaisueper 👍

**Pull request merged**